DEFAULT_CONFIG.json5

```diff
@@ -413,25 +413,15 @@
               /// The maximum time in microseconds to wait for an available batch before dropping a droppable message if still no batch is available.
               wait_before_drop: 1000,
               /// The maximum deadline limit for multi-fragment messages.
-              /// When sending multi-fragment message, for each consecutive fragment the deadline from
-              /// the start point of message transmission is calculated as 
-              /// wait_before_drop * 2 ^ min(N, ttl) where N is a fragment number.
-              /// Thus, this parameter allows setting a limitation for maximum time for multi-fragment
-              /// message transmission as wait_before_drop * 2 ^ ttl
-              ttl: 8,
+              max_wait_before_drop_fragments: 255000,
```

Comment:
255 milliseconds seems a bit too much to me. What about a less aggressive 50 ms?

Reply:
This is just the same as previous `ttl: 8`
Ok, let it be 50